### PR TITLE
Add missing filters info and corrected info on some modules

### PR DESCRIPTION
**Pull request by TheRealPear**
This pull request adds some missing information concerning the `pulse`, `countdown`, `after`, and `players` filter. Additionally, I corrected some information regarding Ender chests and relabeled the values for some items (formatted message instead of strings, etc.) and some editorial changes.

### CODE REVIEW

File: docs/modules/blocks/enderchests.mdx [hunk 1]
```diff
@@ -85,7 +84,7 @@ Additionally, players cannot place Ender chests under any circumstances.
           Region in which a player's Ender chest content will be dropped off.
         </td>
         <td>
-          <a href="/docs/modules/mechanics/regions">Region</a>
+          <a href="/docs/modules/mechanics/regions">Bounded Region</a>
```
Comment (Pablete1234) [inline docs/modules/blocks/enderchests.mdx:87]: Needs to be "Randomize-able" Region, not all bounded regions will work.
As an example, a sphere region (which is bounded) won't work.

File: docs/modules/blocks/enderchests.mdx [hunk 2]
```diff
@@ -157,7 +156,7 @@ Additionally, players cannot place Ender chests under any circumstances.
           The region the drop-off applies to.
         </td>
         <td>
-          <a href="/docs/modules/mechanics/regions">Bounded Regions</a>
+          <a href="/docs/modules/mechanics/regions">Bounded Region</a>
```
Comment (Pablete1234) [inline docs/modules/blocks/enderchests.mdx:159]: given that the region & filter are defined as properties, there's no reason to re-repeat the same thing in a second table. the whole "Dropoff Sub-elements" table can go

File: docs/modules/mechanics/actions-triggers.mdx [hunk 3]
```diff
@@ -493,7 +533,15 @@ trigger an Action.
...
         </td>
-        <td>Sets an Action.</td>
+        <td>
+          <span
+            className="badge badge--secondary"
+            title="Can be either this attribute or a sub-element."
+          >
+            Property
+          </span>
+          Sets a previously defined action.
```
Comment (Pablete1234) [inline docs/modules/mechanics/actions-triggers.mdx:543]: *previously defined* made sense when you could only reference by id, maybe this text can be tweaked
`Sets the action to run when the filter allows`

File: docs/modules/mechanics/filters.mdx [hunk 4]
```diff
@@ -1216,11 +1292,15 @@ This filter is a [Dynamic Filter](#dynamic-filters).
 
 ```
 <filters>
-    <!-- Pulses activation for 15 seconds in a 60 second time period -->
+    <!-- Pulses activation for 15 seconds in a 60 seconds time period -->
```
Comment (Pablete1234) [inline docs/modules/mechanics/filters.mdx:1295]: Pretty sure the former was correct? "60 second time period" sounds more correct than with the s

File: docs/modules/blocks/enderchests.mdx (hunk 2 above)
Comment (TheRealPear) [reply docs/modules/blocks/enderchests.mdx:159]: This is mostly precedent from other pages having a similar layout. [Block Drops](https://pgm.dev/docs/modules/blocks/blockdrops), [Falling Blocks](https://pgm.dev/docs/modules/blocks/falling-blocks), [Portals](https://pgm.dev/docs/modules/mechanics/portals)

Comment (Pablete1234) [reply docs/modules/blocks/enderchests.mdx:159]: i thought the standard was to list them just in the attributes region with a "property" tag, if it seems more common to list them in both places then that's fine, but i think some other places will need expansion as not all attributes marked as property have another table for the children

Comment (TheRealPear) [reply docs/modules/blocks/enderchests.mdx:159]: I'll remove it from the enderchest page for now and clean up the rest of the pages in a future pull request

Comment (TheRealPear) [reply docs/modules/blocks/enderchests.mdx]: image

**Comment by TheRealPear** [comment]
Just a last minute change here, I have no idea how soon this will be reviewed. I've updated Fandom links, once again, to the new Minecraft wiki (https://minecraft.wiki/), which will be the new home for Minecraft information moving forward, with less obstructive ads and better experience for visitors.

I've rebased and unified the existing commits into one for easier reviewing. Let me know if there is anything that needs correcting

File: docs/modules/blocks/enderchests.mdx [hunk 5]
```diff
@@ -9,12 +9,11 @@ The `<enderchest>` tag can contain any number of `<dropoff>` tags. Each `<dropof
 determine where and when items should be dropped from the Ender chest.
 
 Dropoffs handle taking a player's Ender chest inventory and dropping it on the ground at a defined region or it can completely erase it.
-This can be triggered with a filter or once the player switches team or leaves the server.
+This is triggered once the player switches team or leaves the server.
```
Comment (Pablete1234) [inline docs/modules/blocks/enderchests.mdx:12]: Technically if its they leave the match, or "stop participating" in the match (eg: /obs), but its quite a small detail i think most people wont notice the difference

Comment (TheRealPear) [reply docs/modules/blocks/enderchests.mdx:12]: Reworded to `This is triggered once the player switches teams or stops participating in the match.`

**Review by Pablete1234 — APPROVED** [review]

**Comment by CoWinkKeyDinkInc** [comment]
Thanks for changing the wiki links.

**Review by CoWinkKeyDinkInc — APPROVED** [review]

**Pull request merged**